test/ably/resthttp_test.py

```diff
@@ -135,3 +137,14 @@ def test_custom_http_timeouts(self):
         self.assertEqual(ably.http.http_open_timeout, 8)
         self.assertEqual(ably.http.http_max_retry_count, 6)
         self.assertEqual(ably.http.http_max_retry_duration, 20)
+
+    def test_version_headers(self):
+        # RSC7a, RSC7b
+
+        req = Request('GET', '/', skip_auth=True)
+
+        self.assertIn('X-Ably-Version', req.headers.keys())
+        self.assertIn('X-Ably-Lib', req.headers.keys())
+
+        self.assertEqual(req.headers.get('X-Ably-Version'), __api_version__)
```

Comment:
I don't like this test because someone changing `version.py` mistakenly will never break any tests, which is partly why tests exist to prevent unexpected regressions.  Instead, I would prefer that you explicitly check for a `0.9` value match for `X-Ably-Version` and a `0.9.*` match for the lib so that until we migrate to `0.10`, we the tests will pass with minor updates.